src/wallet/rpcwallet.cpp

```diff
@@ -3369,13 +3369,13 @@ UniValue rescanblockchain(const JSONRPCRequest& request)
                 "\nRescan the local blockchain for wallet related transactions.\n",
                 {
                     {"start_height", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "0", "block height where the rescan should start"},
-                    {"stop_height", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "tip height", "the last block height that should be scanned"},
+                    {"stop_height", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "", "the last block height that should be scanned. If none is provided it will rescan up to the tip."},
```

Comment:
nit, to the tip at call time, not return time.

Reply:
No, at *return time*, actually

Reply:
Err sorry, I messed it 🤦‍♂️I meant that.

Reply:
Thanks, took your suggestion.